pom.xml

```diff
@@ -135,6 +135,13 @@
           <artifactId>spring-boot-maven-plugin</artifactId>
           <version>${version.org.springframework.boot}</version>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-deploy-plugin</artifactId>
+          <configuration>
+            <skip>true</skip>
```

Comment:
Keep this but add skip=false to distribution module. The sources distribution artifact is consumed in product build.